### PR TITLE
Goat index

### PR DESCRIPTION
**Pull request by FFrch**
GoatsController with its different actions (CRUD)
Views relates to this action with minimum information displayed inside.

Waiting for seeds to me add to origin master to improve available information is views on create link_to / anchor between the differents pages views.

### CODE REVIEW

File: app/controllers/goats_controller.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    @goat = Goat.find(params[:id])
+  end
+
+  def new
+    @goat = Goat.new
+  end
+
+  def create
+    @goat = Goat.new(goat)
+    @goat.save
```
Comment (sgaignard) [inline app/controllers/goats_controller.rb:16]: Besoin d'associer un goat a un utilisateur...`goat.user = @user` ou un truc dans le genre...

**Review by sgaignard — COMMENTED** [review]
See comments for changes ;)

File: app/controllers/goats_controller.rb (hunk 1 above)
Comment (FFrch) [reply app/controllers/goats_controller.rb:16]: I added => @goat.user = @current_user

File: app/controllers/goats_controller.rb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  def destroy
+    @goat = Goat.find(params[:id])
+    @goat.destroy
+    redirect_to goats_path
+  end
+
+  private
+
+  def goat_params
+    params.require(:goat).permit(:name, :color, :cuteness, :price)
```
[the remark this answers is not in the document]
Comment (FFrch) [reply app/controllers/goats_controller.rb:41]: Not mandatory

**Pull request merged**